src/externs.ts

```diff
@@ -451,6 +451,25 @@ export function generateExterns(
     }
   }
 
+  function writeExportDeclaration(
+      exportDeclaration: ts.ExportDeclaration, namespace: ReadonlyArray<string>) {
+    if (!exportDeclaration.exportClause) {
+      emit(`\n// TODO(tsickle): export * declaration in ${
+          locationStr(exportDeclaration, namespace)}`);
+      return;
+    }
+    for (const exportSpecifier of exportDeclaration.exportClause.elements) {
+      if (!exportSpecifier.propertyName) {
+        emit(`\n// TODO(tsickle): unnamed export of ${exportSpecifier.name.text} in ${
+            locationStr(exportDeclaration, namespace)}`);
+        continue;
+      }
+      emit('/** @const */\n');
+      writeVariableStatement(
+          exportSpecifier.name.text, namespace, exportSpecifier.propertyName.text);
```

Comment:
Will this do the wrong thing when name == propertyName?  Or is propertyName null in that case and we've already bailed out above?
Maybe add an "export {foo}" to the test just to show it?

Reply:
Done.